.travis.yml

```diff
@@ -29,6 +29,10 @@ before_install:
   - sudo make install
   - cd ..
   - date
+  - git clone https://github.com/apache/parquet-format.git
```

Comment:
Since the travis update is on master, I can cherry-pick that commit and put this "hack" to branch specific script.

Reply:
Please, also keep in mind that this branch is 41 commits behind master currently. 1 of this 41 is the column indexes feature which was quite huge. I would suggest getting these changes to this feature branch as soon as possible or at least check how much work does the merge would require.

Reply:
The current PR seems fine only that the travis build was not re-triggered after you re-targeted to `bloom-filter`. If you send an additional commit (even a whitespace or something) it will trigger travis and we'll see if everything is fine.